.github/workflows/scenario-report.yaml

```diff
@@ -1,16 +0,0 @@
-name: Scenario Report
-on:
-  workflow_run:
-    workflows: ['Run Scenarios']  # runs after CI workflow
-    types:
-      - completed
-jobs:
-  scenario-report:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: dorny/test-reporter@v1
-      with:
-        artifact: scenario-results        # Artifact name
-        name: Scenario Tests              # Name of the check run which will be created
-        path: 'scenario-results.json'     # Path to test results (inside artifact .zip)
-        reporter: mocha-json              # Format of test results
```

Comment:
So we didn't need these?

Reply:
I don't think we do, since we are actually running the same exact step at the end of `run-scenarios.yaml` (and similarly, in `run-unit-tests.yaml`).